tests/nightly/test_large_array.py

```diff
@@ -27,7 +27,7 @@
 
 from mxnet.test_utils import rand_ndarray, assert_almost_equal, rand_coord_2d, default_context, check_symbolic_forward, create_2d_tensor
 from mxnet import gluon, nd
-from common import with_seed, with_post_test_cleanup
```

Comment:
Why remove this? It attempts to free up memory after each test passes. Its good to keep it in order to avoid OOM on CI instances.

Reply:
I think tt's unused currently? You can search for with_post_test_cleanup in the source code and there's no place it's called

If we need this functionality, I suggest to implement it via pytest API in the `conftest.py` file.